js_modules/dagster-ui/packages/ui-core/src/hooks/useThrottledMemo.tsx

```diff
@@ -29,16 +29,30 @@ export const ThrottledMemoBatchingContext = createContext<{enqueue: (update: ()
 );
 
 export const useThrottledMemo = <T,>(
-  factory: () => T,
+  factoryArg: () => T,
   deps: React.DependencyList,
-  delay: number,
+  throttleDelay?: number,
 ): T => {
+  const delayRef = useRef(32); // 32ms = 30fps - Only used if throttleDelay is not passed in
+  const factory = useCallback(() => {
+    const start = Date.now();
+    const res = factoryArg();
+    const end = Date.now();
+
+    // Multiply by 4 so that if this is slow to compute we don't use up all of the wall time.
+    delayRef.current = Math.max((end - start) * 4, 32);
```

Comment:
I could probably jack this up even higher. Maybe 6? I don't see much downside to doing it, it would just make the page even more responsive.

Reply:
Might be worth pulling these numbers out into named consts and adding some context with comments -- it's a bit hard to understand why these were the values chosen.

Reply:
+1 for names here that inline your explanations - maybe `const MIN_DELAY_30FPS = 32` and `const PREFERRED_DELAY_MULTIPLIER = 4` (or something? that one is kinda hard to name)